client-generated/package.json

```diff
@@ -12,7 +12,7 @@
   "typings": "./dist/index.d.ts",
   "scripts": {
     "build": "rimraf dist/ && tsc",
-    "prepare": "npm run build",
+    "postinstall": "npm run build",
```

Comment:
When running `npm install`, our client automatically gets built. When I looked at the source code, I was searching for a postinstall script, but did not find one. As it turns out, I used `prepare` instead of `postinstall`, which is a pretty similar life cycle hook. But to make things clearer, I decided to actually switch to `postinstall`, because that is probably what developers expect here.